sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/CryptographyClient.java

```diff
@@ -131,21 +97,18 @@ public EncryptResult encrypt(EncryptionAlgorithm algorithm, byte[] plaintext, by
      * <p><strong>Code Samples</strong></p>
      * <p>Encrypts the content. Subscribes to the call asynchronously and prints out the encrypted content details when
      * a response has been received.</p>
-     * {@codesnippet com.azure.security.keyvault.keys.cryptography.CryptographyClient.encrypt#EncryptionAlgorithm-byte-byte-byte-Context}
+     * {@codesnippet com.azure.security.keyvault.keys.cryptography.CryptographyClient.encrypt#EncryptionAlgorithm-byte-Context}
      *
      * @param algorithm The algorithm to be used for encryption.
      * @param plaintext The content to be encrypted.
-     * @param iv The initialization vector
-     * @param authenticationData The authentication data
      * @param context Additional context that is passed through the Http pipeline during the service call.
      * @return A {@link EncryptResult} whose {@link EncryptResult#getCipherText() cipher text} contains the encrypted
      * content.
      * @throws ResourceNotFoundException if the key cannot be found for encryption.
      * @throws NullPointerException if {@code algorithm} or  {@code plainText} is null.
```

Comment:
Does this also return a `UnsupportedOperationException` ?
nit: extra space

Reply:
That exception is tied more to the type of key configured in the builder and the key operation allowed on it by the user.
Not tied to any of the parameters of the API.
Best to document it at client level.

Reply:
On Second thought, methods will work better compared to Client level.
Add it at method level.